src/config/backends/config-fs.ts

```diff
@@ -117,6 +117,7 @@ export class ConfigFs extends ConfigBackend {
 	}
 
 	private async writeConfigJSON(config: ConfigfsConfig): Promise<void> {
+		console.log(this.ConfigFilePath);
```

Comment:
```suggestion
```

I usually do a search for console.log before submitting to find any debugging code :p

Reply:
😳 thanks!